frontend/src/components/translation/TranslationTable.tsx

```diff
@@ -66,13 +132,54 @@ const TranslationTable = ({
           </Text>
         )}
         <Flex direction="column" width="130px" margin="10px">
-          <Badge
-            textTransform="capitalize"
-            variant={getStatusVariant(storyLine.status)}
-            marginBottom="10px"
-          >
-            {storyLine.status}
-          </Badge>
+          {!editable ? (
+            <Menu>
+              <MenuButton
+                as={Badge}
+                textTransform="capitalize"
+                variant={getStatusVariant(storyLine.status)}
+                marginBottom="10px"
+              >
+                {storyLine.status}
+                <ChevronDownIcon />
+              </MenuButton>
+              <MenuList>
+                <MenuItem
+                  icon={<CheckIcon />}
+                  onClick={async () => {
+                    handleStatusChange(storyLine, "APPROVED");
+                  }}
+                >
+                  Approve
+                </MenuItem>
+                <MenuItem
```

Comment:
remove this: You should never be able to set the status to "Action Required" manually. Leaving a comment will automatically change the state and take care of this.